app/views/org_admin/templates/_funder_templates_list.html.erb

```diff
@@ -0,0 +1,106 @@
+<% # locals: templates, current_org %>
+
+<div class="table-responsive">
+  <table class="table table-hover table-bordered tablesorter">
+    <thead>
+      <% if scope.length > TABLE_FILTER_MIN_ROWS %>
+        <tr>
+          <th colspan="5" class="sorter-false">
+            <%= render(partial: "shared/table_filter", locals: { placeholder: _('Search')}) %>
+          </th>
+        </tr>
+      <% end %>
+      <tr>
+        <th><%= _('Template Name') %></th>
+        <th><%= _('Funder') %></th>
+        <th><%= _('Status') %></th>
+        <th><%= _('Edited Date') %></th>
+        <th class="sorter-false"></th>
+      </tr>
+    </thead>
+    <tbody>
+    <% scope.each do |template| %>
+      <% customization = customizations.select{ |t| t.customization_of == template.dmptemplate_id }.first %>
+      <tr>
+        <td>
+          <%= template.title %>
+        </td>
+          <td>
+            <%= raw template.org.name %>
+          </td>
+          <td>
+            <% if current_user.can_super_admin? %>
+              <%= (template.published? ? _('Published') : _('Unpublished')) %>
+            <% else %>
+              <% if customization.present? %>
+                <!-- If the original funder template has been changed -->
+                <% if customization.updated_at < template.updated_at %>
+                  <%= _('Original funder template has changed!')%>
+                <% elsif !template.published? %>
+                  <!-- The template does not have a live version -->
+                  <%= b_label = _('Funder version is un-published') %>
+                <% elsif customization.dirty? %>
+                  <%= _('You have un-published changes') %>
+                <% else %>
+                  <%= _('Published') %>
+                <% end %>
+              <% else %>
+                <%= _('Not Customised') %>
+              <% end %>
+            <% end %>
+          </td>
+          <td>
+            <% last_temp_updated = template.updated_at %>
+            <%= l last_temp_updated.to_date, formats: :short %>
+          </td>
+          <td>
+            <div class="dropdown<%= (!current_user.can_super_admin? || (current_user.can_super_admin? && current_org == template.org.id)) ? '' : ' hide'%>">
```

Comment:
Do we refresh the page when the user switches organization?
If so I propose not-rendering the drop downs instead of rendering them as hidden unless we un-hide them from the JS somewhere

Reply:
no, its an ajax call with some JS to handle which Orgs' publish/edit/etc. links are visible

Reply:
Ok, this comment is invalid then.